Project.toml

```diff
@@ -13,9 +13,9 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 [compat]
 Distances = "^0.9,^0.10"
 MLJModelInterface = "^0.3.5,^0.4, 1.0"
-MultivariateStats = "0.9"
+MultivariateStats = "0.9.1"
 StatsBase = "0.32, 0.33"
-julia = "1"
+julia = "1.1"
```

Comment:
This might as well be 1.6, because MLJBase now only supports 1.6 and above, and 1.6 is the Julia official Long Term Support release.